src/main/java/com/capitalone/dashboard/client/RestClient.java

```diff
@@ -153,13 +152,16 @@ public RestClient(Supplier<RestOperations> restOperationsSupplier) {
 
         long start = System.currentTimeMillis();
         ResponseEntity<String> response;
-        HttpStatus status = null;
+        String status = null;
         try {
             HttpEntity entity = headers==null?null:new HttpEntity(headers);
             response = restOperations.exchange(url, HttpMethod.GET, entity, String.class);
-            status = response.getStatusCode();
+            status = response.getStatusCode().toString();
```

Comment:
Can we make this null safe ? `(response==null) ? "null : String.valueOf(response.getStatusCode())"`